docs/about/themes-translation.md

```diff
@@ -146,61 +148,15 @@ thank you!
 
 ## Updating theme localization files
 
-If you changed some text in the theme themselves and modified the HTML source
-of a theme, it is necessary to follow the steps below:
+If a [theme's `messages.pot` template file has been updated](/user-guide/custom-themes/#localizing-themes)
+since the `messages.po` was last updated for your locale, follow the steps below
+to update the theme's `messages.po` file:
 
-1. [Extract each theme's text](#extracting-text-from-themes) to update their
-Portable Object Template files
-2. [Update the translation catalog `messages.po` files](#updating-the-translation-catalogs)
-for every supported locale
```

Comment:
Sorry if I wasn't clear. But I expect this step (updating po files) to be done by the translator as their first step.

The theme dev would update the `pot`. The translator updates the `po` file, first by running `update_catalog` and then by editing the translations. We should never have a `po` file with empty/inaccurate translations it in, which is exactly what we would get if the theme dev ran `update_catalog`.

The tricky part about this is that we could end up with a mismatch between the theme and the `po` files for some locales. However, this should be easy to detect if the last modified date of a `po` file is earlier than the `pot` file for that theme. I wonder how that will affect our tests though. We may need to revisit this.